test/testMethods.ts

```diff
@@ -333,4 +333,13 @@ module TestMethods {
       return "#" + redHex + greenHex + blueHex;
   }
 
+  export function assertWarns(funct: Function, warningMessage: string, assertMessage: string) {
```

Comment:
I prefer `function` or `fn` instead of `funct`. And then, if possible, invoke it with `funct.call(this)` instead of `funct()`. But this is just a nitpick